scripts/dev.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# local development start script to prepare dev related env
+
+SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+source "$SCRIPTS_DIR/shellUtils.sh"
```

Comment:
Just wondering do we need this?

Reply:
The utils are mainly to get color output, though I'm not using them 

I've kept the line because I thought if we have more scripts that would be loaded in `dev.sh`  
Instead of every script having to do 

```
SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
source "$SCRIPTS_DIR/shellUtils.sh"
```

They'll just be able to use the utils directly

There's no harm but we can remove it since it's not really used